File: AppGestaoCursos/src/app/models/aluno.ts
import { Curso } from "./curso";

export class Aluno {
  id: string;
  nome: string;
  data_Nascimento: string;
  email: string;
  curso: Curso;
  ativo: boolean;
  data_Criacao: string;
  data_Atualizacao: string;
}

File: ApiGestaoCursos/ApiGestaoCursos.Domain/Entities/Aluno.cs
namespace ApiGestaoCursos.Domain.Entities
{
    public class Aluno : Base
    {
        public string? Nome { get; set; }
        public DateTime Data_Nascimento { get; set; }
        public string? Email { get; set; }
        public Guid? CursoId { get; set; }
        public Boolean Ativo { get; set; }
        public Curso? Curso { get; set; }
    }
}
